hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java

```diff
@@ -120,23 +124,27 @@ public boolean processWithLegacy(OMUpdateEventBatch events) {
         if (!updatedKeyInfo.getKeyName().endsWith(OM_KEY_PREFIX)) {
           switch (action) {
           case PUT:
-            handlePutKeyEvent(updatedKeyInfo, nsSummaryMap);
+            handlePutKeyEvent(updatedKeyInfo, nsSummaryMap,
+                new HashMap<>(), 1L);
```

Comment:
- There can be some db event if not passed as null, like delete may check db for exist, ...
- earlier seems this can be null, but not with this does not seems this map will be null

Reply:
Ok, tested by passing null for LegacyTask. Works fine.